File: 백준/Silver/2563. 색종이/색종이.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    
    private static final BufferedReader br = new BufferedReader(
        new InputStreamReader(System.in)
    );

    private static boolean[][] Canvas = new boolean[100][100];

    public static void main(String[] args) throws IOException {
        int numOfPapers = Integer.parseInt(br.readLine().split(" ")[0]);

        String[] lineString;
        int count = 0;

        int coord_x, coord_y;

        for (int index = 0; index < numOfPapers; index++) {
            lineString = br.readLine().split(" ");
            
            coord_x = Integer.parseInt(lineString[0]) - 1;
            coord_y = Integer.parseInt(lineString[1]) - 1;
            
            for (int i = coord_x; i < coord_x + 10; i++) {
                for (int j = coord_y; j < coord_y + 10; j++) {
                    if (Canvas[i][j])   continue;

                    Canvas[i][j] = true;
                    count++;
                }
            }
        }
        
        System.out.println(count);
    }
}

File: 백준/Silver/2563. 색종이/색종이.kt
import java.io.BufferedReader
import java.io.IOException
import java.io.InputStreamReader
import kotlin.jvm.Throws

@Throws(IOException::class)
fun main(args: Array<String>) = with(BufferedReader(InputStreamReader(System.`in`))) {
    val canvas = Array(100) { BooleanArray(100) }

    val numOfPapers = readLine().split(" ".toRegex()).dropLastWhile { it.isEmpty() }
        .toTypedArray()[0].toInt()

    var lineString: Array<String>
    var count = 0

    var coord_x: Int
    var coord_y: Int

    for (index in 0 until numOfPapers) {
        lineString =
            readLine().split(" ".toRegex()).dropLastWhile { it.isEmpty() }.toTypedArray()

        coord_x = lineString[0].toInt() - 1
        coord_y = lineString[1].toInt() - 1

        for (i in coord_x until coord_x + 10) {
            for (j in coord_y until coord_y + 10) {
                if (canvas[i][j]) continue

                canvas[i][j] = true
                count++
            }
        }
    }

    println(count)
}